README_KO.md

```diff
@@ -0,0 +1,34 @@
+# RMS의 지지를 위한 공개 서한.
+
+서명을 위해 `_data/signed/` 폴더 안에 `<username>.yaml`라는 이름의 파일을 만들어야 합니다.
+다음과 같은 내용이 포함되어 있어야 합니다:
+
+```yaml
+name: <your name here>
+link: <link to your profile or site>
+```
+
+`<>`는 제외됩니다..
+
+예시:
+```yaml
+name: Example name
+link: https://example.com/
+```
+
+파일에 `<>`를 사용하지 말아 주십시오, 비 아스키(non-ascii) 문자를 파일 이름에 사용하지 말아주십시오.
+가능하다면 실명을 사용해 주시고, 괄호 안에 프로젝트와 소속 기관을 작성해주십시오.
+
+작성이 완료 되었다면, pull request를 작성해주십시오.
+
+우리의 목소리를 굳건하면서도 전문적으로 유지합시다.
+
+가능하다면 본 공개서한을 포럼이나 SNS에 공유해 주시고, 이 문제에 대해 도움이 될만한 기자들에게 알려주십시오.
+
+**Pull requests 는 12시간 내에 merge 됩니다 - 많은 양의 PR에 따라, PR들은 일괄적으로 처리됩니다**
+
+## Chatrooms
+
+- **Matrix.org:** #free-rms:matrix.org
+- **IRC:** #free-rms at chat.freenode.net
```

Comment:
Can you please translate the `at` preposition too?

Reply:
Okay

Reply:
Done